app/views/occupation_standards/_ojt_accordion.html.erb

```diff
@@ -12,13 +12,15 @@
                     <div class="flex items-center">
                         <div class="flex flex-col md:flex-row">
                             <% unless work_process.competencies.count.zero? %>
-                            <div class="m-1.5 w-28 rounded bg-yonder-100 px-1 py-3 text-center">
-                                <div class="text-3xl font-bold text-yonder-600"><%= work_process.competencies.count %></div>
-                            </div>
+                                <div class="m-1.5 w-28 rounded bg-yonder-100 px-1 py-3 text-center">
+                                    <div class="text-3xl font-bold text-yonder-600"><%= work_process.competencies.count %></div>
+                                </div>
+                            <% end %>
+                            <% unless work_process.hours.nil? %>
```

Comment:
I really like `if` over `unless` when possible, can this be: `if work_process.hours.present?` ?